recipes/mountbroker.rb

```diff
@@ -0,0 +1,44 @@
+#
+# Cookbook Name:: gluster
+# Recipe:: mountbroker
+#
+# Copyright 2016, Yakara Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+include_recipe 'gluster::geo_replication_install'
+require 'json'
+
+mb = node['gluster']['mountbroker']
+
+group mb['group'] do
```

Comment:
I've been using Chef a long time but I reserve the right to be completely wrong. :wink: I decided to test this theory by temporarily adding the same group in geo_replication_install as I honestly wasn't sure. It actually doesn't blow up.

```
       Recipe: gluster::geo_replication_install
         * yum_package[glusterfs-geo-replication] action install (up to date)
         * group[geogroup] action create
           - create group geogroup
       Recipe: gluster::mountbroker
         * group[geogroup] action create (up to date)
```

What I was even less sure about was whether `system false` in the first resource would take precedence over `system true` in the second. I was pleased to find that it does, which is helpful. I guess that's why we have `edit_resource` aka `chef-rewind`.